app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :authenticate_user!
+
+  def index
+    @users = policy_scope(User)
+    authorize User
+    @roles = Role.all
+    @current_user = current_user
```

Comment:
I do not think you need this variable

Reply:
you can use `curreny_user` directly in the view

Reply:
Will be fixed